execution.py

```diff
@@ -494,6 +494,7 @@ def execute(self, prompt, prompt_id, extra_data={}, execute_outputs=[]):
                 result, error, ex = execute(self.server, dynamic_prompt, self.caches, node_id, extra_data, executed, prompt_id, execution_list, pending_subgraph_results)
                 if result == ExecutionResult.FAILURE:
                     self.handle_execution_error(prompt_id, dynamic_prompt.original_prompt, current_outputs, executed, error, ex)
+                    self.success = False
                     break
                 elif result == ExecutionResult.PENDING:
                     execution_list.unstage_node_execution()
```

Comment:
Missing a: self.success = True when it succeeds.

Reply:
Done